<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="2" parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="2" parallel="tests" name="TestRunner-2(failed)">
    <classes>
      <class name="com.vtiger.CreateOpportiunityTC1">
        <methods>
          <include name="createOpportiunity"/>
          <include name="beforesuite"/>
          <include name="afterSuite"/>
          <include name="closeBrowser"/>
          <include name="logoutfromApp"/>
          <include name="launchBrowser"/>
          <include name="logintoapp"/>
        </methods>
      </class> <!-- com.vtiger.CreateOpportiunityTC1 -->
    </classes>
  </test> <!-- TestRunner-2(failed) -->
  <test thread-count="2" parallel="tests" name="TestRunner-1(failed)">
    <classes>
      <class name="com.vtiger.CreateOpportunityUsingOrgErasebuttonTC5">
        <methods>
          <include name="beforesuite"/>
          <include name="createOppusingOrgErasebutton"/>
          <include name="afterSuite"/>
          <include name="closeBrowser"/>
          <include name="logoutfromApp"/>
          <include name="launchBrowser"/>
          <include name="logintoapp"/>
        </methods>
      </class> <!-- com.vtiger.CreateOpportunityUsingOrgErasebuttonTC5 -->
    </classes>
  </test> <!-- TestRunner-1(failed) -->
</suite> <!-- Failed suite [Suite] -->
